frontend/admin/src/js/lang/fr.json

```diff
@@ -1664,5 +1664,599 @@
+  "+OSYz7": {
+    "defaultMessage": "Nouvel utilisateur",
+    "description": "Text label for link to create new user on admin table"
+  },
+  "+svnC6": {
+    "defaultMessage": "Publier",
+    "description": "Heading for the publish pool dialog"
+  },
+  "+umNAP": {
+    "defaultMessage": "Créer un nouveau bassin",
+    "description": "Header for page to create pool advertisements"
+  },
+  "/6voUd": {
+    "defaultMessage": "Modifier l'annonce du bassin",
+    "description": "Header for page to edit pool advertisements"
+  },
+  "/7tcPl": {
+    "defaultMessage": "Votre travail (anglais)",
+    "description": "Title for English pool advertisement Work"
+  },
+  "/QUq6L": {
+    "defaultMessage": "Vous êtes certain de vouloir continuer?",
+    "description": "Second paragraph for Delete pool dialog"
+  },
+  "044hi7": {
+    "defaultMessage": "Copier le lien",
+    "description": "Button text to copy a url"
+  },
+  "0qKFaP": {
+    "defaultMessage": "Ce bassin est actuellement en cours de publication et ouvert à la réception de nouvelles candidatures.",
+    "description": "Status description of a pool in PUBLISHED status"
+  },
+  "1HPhji": {
+    "defaultMessage": "Filtres",
+    "description": "Text label for button to open filter dialog on admin tables."
+  },
+  "2++hVA": {
+    "defaultMessage": "Vous êtes certain de vouloir continuer?",
+    "description": "Third paragraph for Close pool dialog"
+  },
+  "2asU3k": {
+    "defaultMessage": "Enregistrer les compétences essentielles",
+    "description": "Text on a button to save the pool essential skills"
+  },
+  "36k+Da": {
+    "defaultMessage": "Nom",
+    "description": "Label for user table search dropdown (name)."
+  },
+  "3mrTn5": {
+    "defaultMessage": "Prolonger la date de clôture",
+    "description": "Heading for the extend pool closing date dialog"
+  },
+  "45BhQw": {
+    "defaultMessage": "Vous êtes sur le point de PUBLIER ce bassin.",
+    "description": "First paragraph for publish pool dialog"
+  },
+  "4yBTfg": {
+    "defaultMessage": "<heavyPrimary>Ébauche</heavyPrimary>",
+    "description": "Status name of a pool in DRAFT status"
+  },
+  "5PIt8V": {
+    "defaultMessage": "Sélectionnez la classification prévue pour ce processus de recrutement.",
+    "description": "Helper message for selecting a classification for the pool"
+  },
+  "61XRUJ": {
+    "defaultMessage": "Clé",
+    "description": "Label displayed on the pool form name key field."
+  },
+  "66MUMB": {
+    "defaultMessage": "Enregistrer les autres exigences",
+    "description": "Text on a button to save the pool other requirements"
+  },
+  "6BD4FK": {
+    "defaultMessage": "Votre impact (français)",
+    "description": "Title for French pool advertisement impact"
+  },
+  "6InelL": {
+    "defaultMessage": "Basculer vers filtrer par",
+    "description": "Title to toggle sorting order of a table"
+  },
+  "6l1+3n": {
+    "defaultMessage": "résultats des compétences, recherche par mot-clé"
```

Comment:
Is it possible to add a description to the English string so that one can find the English string if necessary?